composer.json

```diff
@@ -4,7 +4,7 @@
     "description": "PHP5+ Database ORM",
     "keywords": ["orm", "database"],
     "homepage": "http://www.doctrine-project.org",
-    "license": "LGPL",
+    "license": "LGPL-2.1-only",
```

Comment:
I think this can be "LGPL-2.1-or-later" based on this text in the LICENSE file:

>This library is free software; you can redistribute it and/or
    modify it under the terms of the GNU Lesser General Public
    License as published by the Free Software Foundation; either
version 2.1 of the License, or (at your option) any later version.